### ISSUE TITLE
FakeStub docstring triggers syntax warning

### ISSUE THREAD

**Issue by ghost** [marcosschroh/dataclasses-avroschema#485]
**Describe the bug**

The docstring for `FakeStub` contains an invalid escape sequence, `\.`. This will eventually throw an exception in a future Python version.

**To Reproduce**

Running a test suite on Pytest with all warnings enabled and `faker` *not* installed shows this warning:

```
[...]/dataclasses_avroschema/faker.py:8: DeprecationWarning: invalid escape sequence '\.'
```

**Expected behavior**

The code shouldn't trigger any warnings. 

According to [the Python docs](https://docs.python.org/3/library/re.html#module-re), this is technically a bug:

> [...] any invalid escape sequences in Python’s usage of the backslash in string literals now generate a [SyntaxWarning](https://docs.python.org/3/library/exceptions.html#SyntaxWarning) and in the future this will become a [SyntaxError](https://docs.python.org/3/library/exceptions.html#SyntaxError). This behaviour will happen even if it is a valid escape sequence for a regular expression.

This can be fixed by making the docstring a raw string (i.e. prefixing `r`) as if it were a regular expression, like so:

```diff
--- original.py	2023-12-04 08:52:52
+++ fixed.py	2023-12-04 08:52:15
@@ -5,7 +5,7 @@
 except ModuleNotFoundError:  # pragma: no cover
 
     class FakeStub:
-        """A stub for Faker.fake() when the feature [faker] is not enabled.
+        r"""A stub for Faker.fake() when the feature [faker] is not enabled.
         It raises a hard RuntimeError when faker is not installed and
         we tried to invoke `.fake()` on any of our fields.
         We can find the local usages of faker methods (i.e. fake.<method>(*args, **kwargs)) with:
```

**Comment by marcosschroh** [comment]
Do you want to send a PR?
